### PR TITLE
Add working duplicate button

### PR DESCRIPTION
**Pull request by JanekJaniak**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -33,8 +33,13 @@ export default {
     },
 
     duplicateItem(itemID) {
-      console.log(itemID);
-    }
+      const duplicateValue = this.items.find(item => item.id == itemID)
```
Comment (jan-gajdowicz) [inline src/App.vue]: If the name is `duplicateValue` it should return a value

File: src/App.vue [hunk 2]
```diff
@@ -33,8 +33,13 @@ export default {
...
+      const duplicatItem = {
```
Comment (jan-gajdowicz) [inline src/App.vue]: I believe it should be `duplicateItem`

**Review by jan-gajdowicz — CHANGES_REQUESTED** [review]
Couple things

**Review by jasiek-survicate — APPROVED** [review]
GJ! :bicyclist:

**Review by jan-gajdowicz — APPROVED** [review]

**Pull request merged**